.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Linting and Testing
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
```

Comment:
Don't think this will cause any issues but we are using node version lts/iron (currently 20.18.0).